tests/end_to_end/test_docker_monitoring/data/test_cases/cases_test_docker_monitoring.yaml

```diff
@@ -0,0 +1,52 @@
+---
+- name: docker_pull
+  description: Test if an alert is generated when using the `pull` command.
+  configuration_parameters: null
+  metadata:
+    rule.id: '87932'
+    rule.description: 'Docker: Image or repository nginx pulled'
+    extra:
+      data.docker.Action: pull
+      regex: '.+"docker":.+"Action": "(pull)".+"Attributes":.+"(nginx)".+"description": "(Docker: Image or repository nginx pulled)".+'
+  extra_vars:
+    event_description: Run 'docker pull nginx'
+    command: docker pull nginx
+
+- name: docker_run
+  description: Test if an alert is generated when using the `pull` command.
```

Comment:
We are not using `pull` in this test case

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2948/commits/de272050ca3415a878861a8ee0461883c3936f17